<?xml version="1.0"?>

<launch>
    <!-- parameter used to scale simulation velocity -->
    <param name='sim_scale' type='int' value='1'/>

	<include file="$(find assignment3)/launch/sim.launch"></include>
	<include file="$(find assignment3)/launch/gmapping.launch"></include>
    <include file="$(find assignment3)/launch/move_base.launch"></include>
</launch>
